packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -121,9 +122,28 @@ class ApplicationCommand extends Base {
       this.options ??= [];
     }
 
+    if ('default_member_permissions' in data) {
+      /**
+       * The default bitfield used to determine whether this command be used in a guild
+       * @type {?Readonly<PermissionsBitField>}
+       */
+      this.defaultMemberPermissions = data.default_member_permissions
+        ? new PermissionsBitField(BigInt(data.default_member_permissions)).freeze()
+        : null;
+    }
+
+    if ('dm_permission' in data) {
+      /**
+       * Whether the command can be used in dms
+       * @type {boolean}
```

Comment:
This can be undefined given the if check 👀

Reply:
meant to put a warn /info on this, its not present in guild commands ever, think we still document it as ?boolean though